db/seeds.rb

```diff
@@ -125,6 +125,14 @@
 gene2_cells = gene_2.data_arrays.build(name: gene_2.cell_key, array_type: 'cells', cluster_name: expression_file.name,
                                        array_index: 1, study_id: study.id, study_file_id: expression_file.id, values: all_cell_array)
 gene2_cells.save!
+DirectoryListing.create!(name: 'fastq', file_type: 'fastq',
```

Comment:
Nice touch.  Having a directory listing added via synthetic study populator would also be helpful, e.g. for trivially setting up manual testing in the UI.

Reply:
We have a [ticket](https://broadworkbench.atlassian.net/browse/SCP-3061) in the backlog to transition off of `db/seeds.rb` and use `FactoryBot` for seeding test data.  I would argue that whenever we start this work, we can address this more thoroughly.